### ISSUE TITLE
Radial head fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10796]
**Preferred term label:**
Radial head fracture

**Synonyms**

**Definition (free text, please give PubMed ID)**
Fracture of the proximal end of the radius involving the radial head

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Radial head fracture
Capitellum fracture
Monteggia fractures

**Your nano-attribution (ORCID)**

**Comment by MickeySegal** [comment]
This should also include a history of radial head fracture, which we include as a separate finding.  This fracture, even once healed, can result in "Longitudinal instability of the forearm axis" as a result of a radioulnar interosseous membrane tear that becomes evident after the fracture has already healed on imaging.

**Comment by pnrobinson** [comment]
@MickeySegal 
We try to avoid coding too many things in terms because you get an explosion of terms. We would encode a history of the term by either using the term in a "history" slot in software or by corresponding ages in the phenoapcket schema.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001131
- label: Radial head fracture
- parents: [Fractured radius (HP:0003978)](https://hpo.jax.org/app/browse/term/HP:0003978)
- pmid: 
- def: Fracture of the proximal end of the radius involving the radial head.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10796
- synonyms:
